src/renderer/theme.store.ts

```diff
@@ -43,7 +47,7 @@ export class ThemeStore extends Singleton {
   }
 
   @computed get activeTheme(): Theme {
-    return this.themes.get(this.activeThemeId) ?? this.themes.get(defaultTheme);
+    return this.systemTheme ?? this.themes.get(this.activeThemeId) ?? this.themes.get(defaultTheme);
```

Comment:
Are you sure about using nullish coalescing here? Why not simple || ?

Reply:
I would argue that `??` is more correct because the type of `this.systemTheme` is `Theme | undefined`